devicetypes/yooksmart/yooksmart-window-blinds.src/yooksmart-window-blinds.groovy

```diff
@@ -0,0 +1,194 @@
+/**
+ *
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ *
+ *  first release for IKEA smart window blinds
+ */
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+    definition(name: "YookSmart Window Blinds", namespace: "yooksmart", author: "YookSmart", ocfDeviceType: "oic.d.blind", mnmn: "SmartThingsCommunity", vid: "58b24db0-3f12-3cd1-a86f-cc435ae6f991") {
```

Comment:
Please, could You add Your devices to the existing DTH that covers those functionalities ?
https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy

Reply:
Yes, how should I do it?

Reply:
Please create a new pull request (close this one) and edit only zigbee-window-shade-battery.groovy (add Your fingerprints there. Some tweaks in the code might be needed to handle specific configurations for Your devices etc. - for example in  configure() or in parse() methods)